enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,389 @@
+talk over the private network on a high-speed link.
+7. Bootstrapping completes as normal.
+8. The workers will be added using the openshift/[cluster-\*machine-\*] controllers and operators based on the OpenShift
+MAPI model. A Power VS controller needs to be implemented and hooked into the [OpenShift Machine
+Operator][openshift-machine-operator].
+9. The cluster teardown can make use of the existing [go client for Power][power-go-client] via `openshift-install
+destroy cluster`.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer for a
+default size (3 master and 2 worker) cluster.
+- The expectation is that this is fully automated.
+
+### API Extensions
+None
+
+### Implementation Details/Notes/Constraints
+
+#### Existing APIs in the Upstream
+There is a working [implementation of a machine API](https://github.com/openshift-powervs/cluster-api-provider-powervs)
+we've created for testing purposes. Though not ready for upstream, many have been tested and found to be working, such
+as the machine API and node update controller. This was tested with machineset as well, so things look promising.
+
+#### Cloud Controller Manager
+We are planning to reuse the [cloud-provider-ibm][cloud-provider-ibm] for everything except for the node controller. The
+node objects are managed by the cluster-api-provider-powervs node controller.
+
+#### Separate IBM Providers for VPC vs Power VS
+One thing that has been asked and discussed is whether it is possible to create a single abstract IBM provider that
+works with IBM VPC and Power VS. Unfortunately, there are very few similarities between the target environments outside
+of authentication. The APIs, parameters, and image management are all completely different, so we believe the cleanest
+implementation would be a separate provider.
+
+### Risks and Mitigations
+
+The Multi-Arch team has already mitigated much of the risks involved with coordinating a new ppc64le platform. We have
+existing engagements with many of the teams that we will be working with to add this platform. For instance, we
+already have a hardware environment capable of building the OVA images, a dedicated CI team to build out and analyze the
+development-focused testing, and the underlying content release content we'd need to build for ppc64le by the ART team.
+
+## Design Details
+![alt Distinction Between Power VS and IBM Cloud VPC Layers](ibm-cloud-for-power-vs-ipi-overview.png)
+
+The Power Systems Virtual Servers (Power VS) are separated from the rest of the IBM Cloud servers with separate networks
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments. Both public and private network is supported for instances (VMs). For public networks, only
+limited ports are opened for inbound access and this is not configurable.
+
+Basic Walkthrough:
```

Comment:
I really appreciate that you included the level of detail in this walkthrough. It's very helpful to understand who is doing what, and how.